app/src/main/res/layout/activity_u_iscreen.xml

```diff
@@ -0,0 +1,240 @@
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="20dp"
+        android:fontFamily="serif"
+        android:shadowColor="#FFFFFF"
+        android:text="hospital"
+        android:textColor="#4C4545"
+        android:textColorHint="#504A4A"
+        android:textSize="18sp"
+        app:layout_constraintStart_toStartOf="@+id/textgrouper2"
+        app:layout_constraintTop_toTopOf="@+id/textgrouper2" />
+
+    <TextView
+        android:id="@+id/textgrouper1"
+        android:layout_width="390dp"
+        android:layout_height="174dp"
+        android:layout_marginTop="20dp"
+        android:background="#ebfafa"
+        android:textColor="#FFFFFF"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.761"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/QRimage" />
+
+    <TextView
+        android:id="@+id/agetext1"
+        android:layout_width="223dp"
+        android:layout_height="30dp"
+        android:layout_marginStart="28dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="20dp"
+        android:fontFamily="serif"
+        android:text="NULL"
+        android:textColor="#4C4545"
+        android:textSize="18sp"
+        app:layout_constraintBottom_toBottomOf="@+id/textgrouper1"
+        app:layout_constraintEnd_toEndOf="@+id/textgrouper1"
+        app:layout_constraintStart_toEndOf="@+id/agetext"
+        app:layout_constraintTop_toBottomOf="@+id/idtext1" />
+
+    <TextView
+        android:id="@+id/textname"
+        android:layout_width="112dp"
+        android:layout_height="32dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="20dp"
+        android:fontFamily="serif"
+        android:shadowColor="#FFFFFF"
+        android:text="Full Name :"
+        android:textColor="#4C4545"
+        android:textColorHint="#504A4A"
+        android:textSize="18sp"
+        app:layout_constraintStart_toStartOf="@+id/textgrouper1"
+        app:layout_constraintTop_toTopOf="@+id/textgrouper1" />
+
+    <TextView
+        android:id="@+id/nametext1"
+        android:layout_width="223dp"
+        android:layout_height="30dp"
+        android:layout_marginStart="28dp"
+        android:layout_marginTop="20dp"
+        android:fontFamily="serif"
+        android:text="NULL"
+        android:textColor="#4C4545"
+        android:textSize="18sp"
+        app:layout_constraintEnd_toEndOf="@+id/textgrouper1"
+        app:layout_constraintStart_toEndOf="@+id/textname"
+        app:layout_constraintTop_toTopOf="@+id/textgrouper1" />
+
+    <TextView
+        android:id="@+id/idname"
+        android:layout_width="112dp"
+        android:layout_height="27dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="20dp"
+        android:fontFamily="serif"
+        android:shadowColor="#FFFFFF"
+        android:text="ID Document"
```

Comment:
for the text, you should add it to strings.xml and call it here, use this tutorial to lean how : https://www.youtube.com/watch?v=q_gujMR2ggE&ab_channel=BrianFraser